ingestion/src/metadata/data_quality/validations/runtime_param_setter/table_diff_params_setter.py

```diff
@@ -93,19 +114,23 @@ def get_service2_url(self, service1, table2, test_case):
     def get_extra_columns(
         self, key_columns: List[str], test_case
     ) -> Optional[List[str]]:
```

Comment:
Can you add some doc string to explain what we are doing here? 

Same with the other methods